api/views.py

```diff
@@ -194,11 +194,6 @@ def custom_component_controller(request, type_req, user_id, demoid):
     elif type_req == "output":
         model = OutputComponent
         serializer = OutputComponentSerializer
-    else:
```

Comment:
type_req can be either input or output as defined in the url pattern .So this code was redundant .